src/Sylius/Bundle/ShippingBundle/spec/DateAssigner/ShipmentShippedAtDateAssignerSpec.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ShippingBundle\DateAssigner;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\ShippingBundle\DateAssigner\ShipmentShippedAtDateAssignerInterface;
+use Sylius\Component\Shipping\Model\ShipmentInterface;
+
+final class ShipmentShippedAtDateAssignerSpec extends ObjectBehavior
+{
+    function it_implements_an_date_assigner_interface(): void
+    {
+        $this->shouldImplement(ShipmentShippedAtDateAssignerInterface::class);
+    }
+
+    function it_assigns_a_shipped_at_date_to_an_shipment(
```

Comment:
`it_assigns_a_shipping_date_to_the_shipment`? And won't it fit in one line with the argument?

Reply:
you have a greate eye  :D
![Screenshot 2020-02-10 at 14 17 39](https://user-images.githubusercontent.com/29897151/74153114-23adc580-4c10-11ea-9281-c0b820194701.png)